scripts/webpack/webpack.config.ts

```diff
@@ -133,6 +126,9 @@ const webpackConfig: webpack.Configuration = {
       src: paths.packageSrc('react-northstar'),
       faker: 'faker/locale/en',
       'react-hook-form': 'react-hook-form/dist/react-hook-form.ie11',
+      ...(__DEV__ && {
+        'react-dom': '@hot-loader/react-dom',
+      }),
```

Comment:
To get hooks reload working we need to [use patched `react-dom`](https://github.com/gaearon/react-hot-loader#getting-started), but it's required only in dev environment.